aten/src/ATen/core/MetaFallbackKernel.cpp

```diff
@@ -0,0 +1,33 @@
+#include <torch/library.h>
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <c10/core/impl/PyInterpreter.h>
+
+namespace at {
+namespace impl {
+
+static void metaFallback(
```

Comment:
there aren't per-op fallbacks, right? And there isn't really a way to easily register and de-register kernels

Reply:
(technically you can register a boxed function to a single operator) but nvm you're right - the fallback seems like the right thing anyway since you're expecting the python module to eventually register something directly to the operator.